firestore/google/cloud/firestore_v1beta1/bidi.py

```diff
@@ -0,0 +1,559 @@
+# Copyright 2017, Google LLC
```

Comment:
NOTE: this is a copy from pubsub. Plan to refactor to api_core to share.

Reply:
Should that license be 2018, btw?